src/test/java/com/teamrocket/unit/service/DeliveryServiceTest.java

```diff
@@ -182,6 +234,20 @@ void handleOrderReadyEvent() {
                 .findAllByOrderIdAndStatus(orderId, DeliveryStatus.NEW);
     }
 
+    @Test
+    void handleOrderReadyEvent_throwsNoSuchElementException() {
+        assertThrows(NoSuchElementException.class, () -> {
+            sut.handleOrderReadyEvent(-777);
+        }, "No delivery task for order id -777");
```

Comment:
These messages are supposed to provide information about the assertions if the assertion fails. 
What you are writing in all of these strings, are the success senarios as strings, not what may may have gone wrong in the case of failure.

In this case, the message should almost be the opposite. "A delivery task with order id -777 was found, but was not expected